addon-test-support/scroll.js

```diff
@@ -0,0 +1,47 @@
+import getElementWithAssert from './-private/get-element-with-assert';
+import wait from 'ember-test-helpers/wait';
+
+
+/*
+  Triggers a paint (and therefore flushes any queued up scroll events).
+
+  @method triggerFlushWithPromise
+  @return {RSVP.Promise}
+  @private
+*/
+function triggerFlushWithPromise(){
+  return wait().then(() => {
```

Comment:
yeah, that's what I want to play around with. If a user sends a scroll event and we `wait` will that be long enough to trigger it? Not entirely sure.

For our tests, the RAF is actually triggered _by_ the scroll event, so it couldn't possibly be what ends up triggering it. There may be a better way to make sure the event gets triggered though, maybe manually.